src/components/Home/Home.js

```diff
@@ -9,8 +9,13 @@ import PropTypes from 'prop-types';
 
 class Home extends React.Component {
   componentDidMount() {
+    const hourHasPassed =
+      !this.props.latestUsersFetchTime ||
+      Math.abs(Date.now() - this.props.latestUsersFetchTime) > 36e5;
```

Comment:
typo?? 
IIRC `Date.now()` returns UNIX epoch time, so will need to be 
`Math.abs(Date.now() - this.props.latestUsersFetchTime) > 3600000`
(60 * 60 * 1000)